deployment/helm/templates/prometheus.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.prometheus.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: akri-agent-metrics
+  namespace: {{ .Release.Namespace }}
+  labels:
+    release: prometheus
+spec:
+  selector:
+    matchLabels:
+      app: akri-agent-metrics
+  endpoints:
+  - port: metrics
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: akri-controller-metrics
+  namespace: {{ .Release.Namespace }}
+  labels:
+    release: prometheus
+spec:
+  selector:
+    matchLabels:
+      app: akri-controller-metrics
+  endpoints:
+  - port: metrics
+---
+{{- /* If Prometheus is enabled, create a service to target the metrics */}}
```

Comment:
Could we use PodMonitor to not have to create separate services? Or is there a downside to that approach?

https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/design.md#podmonitor

Reply:
Thanks for pointing this out. This would be simpler. I'll try it out.